src/tokens/utils.ts

```diff
@@ -159,7 +160,7 @@ export function getSupportedNetworkIdsForTokenBalances(): NetworkId[] {
     .showBalances
 }
 
-export function getTokenId(networkId: NetworkId, tokenAddress?: string): string {
+export function getTokenId(networkId: NetworkId, tokenAddress?: string | Address): string {
```

Comment:
nit: Do we need this change?
An `Address` is a `string` already, so this should already be covered. And I don't see any specific usage of the `Address` type.
I agree that `tokenAddress` should be always an `Address`, but if we want to do that change, shouldn't we remove  the `string` type?

Reply:
i added this because typescript complained when i passed a type `Address` into this function, but it looks like i didn't need it in the end so i removed this change. thanks for calling it out!